modules/local/nacho/normalize/main.nf

```diff
@@ -1,7 +1,8 @@
 process NACHO_NORMALIZE {
+    tag '$sample_sheet'
     label 'process_single'
 
-    conda "r-nacho=2.0.6 r-dplyr=1.2.2 r-ggplot2=3.4.4 r-fs=1.6.2 r-readr=2.1.5 r-tidyr=1.3.0"
+    conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/mulled-v2-9d27fb90b747ac2e521703d90daacce9cc1f33c5:98395a5d2e19da46499873cd2d76be73d6a0950d-0' :
         'biocontainers/mulled-v2-9d27fb90b747ac2e521703d90daacce9cc1f33c5:98395a5d2e19da46499873cd2d76be73d6a0950d-0' }"
```

Comment:
You can replace these by Seqera containers (not required in this issue but nice to have).

Reply:
@atrigila We were discussing this with @nschcolnicov, not going too far away from the ticket scope. But I can do it now, give me some more time to update it